app/public/style.css

```diff
@@ -0,0 +1,223 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+:root {
+  --backgroundcolor: #edf0ef;
+  --inactiv-element: #a9b6b3;
+
+  --surface: #fffafa;
+  --surface-variante: #f5f5f5;
+
+  --primary: #1a1e22;
+  --primary-variante: #272d33;
+
+  --secondary: #494e54;
+  --secondary-variante: #dce1e0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  color: var(--primary);
+}
+
+.background-color {
+  background-color: var(--backgroundcolor);
+}
```

Comment:
siehe kommentar oben.
Diese .class kannst du dir dann sparen, wenn du den background-color in den body-selector definierst.